crates/sui-graphql-rpc/src/types/query.rs

```diff
@@ -183,6 +187,7 @@ impl Query {
             .extend()
     }
 
+    /// The objects that exist in the network.
```

Comment:
nit: We should use a similar pattern of doc comment for `checkpoints`, `object_connection`, etc, given that they are conceptually similar APIs.

Reply:
Sounds good!